### PR TITLE
feat(account-lib): fix is valid account id

### PR DESCRIPTION
**Pull request by facundoCarreSimtlix**
fix isValidAccountId method

What happened?
If an empty string is sent to isValidAccountId it returns true.

Requeriments:
Should not pass for empty strings ' '.
STLX-16458

### CODE REVIEW

**Review by ipelaez-simtlix — APPROVED** [review]

File: modules/account-lib/src/coin/near/utils.ts [hunk 1]
```diff
@@ -46,14 +46,15 @@ export class Utils implements BaseUtils {
    * or can create a new keyPair
    */
   isValidAccountId(accountId: string): boolean {
-    try {
+    if (
```
Comment (silalvado) [inline modules/account-lib/src/coin/near/utils.ts:49]: You can avoid this "if" and directly return the logical expression

**Review by silalvado — APPROVED** [review]

**Review by ipelaez-simtlix — DISMISSED** [review]

**Review by rliberal-simtlix — DISMISSED** [review]

File: modules/account-lib/src/coin/near/utils.ts [hunk 2]
```diff
@@ -46,16 +46,12 @@ export class Utils implements BaseUtils {
    * or can create a new keyPair
    */
   isValidAccountId(accountId: string): boolean {
-    try {
-      // this regex is from near doc, https://docs.near.org/docs/concepts/account
+    return (
       (/^(([a-z\d]+[\-_])*[a-z\d]+\.)*([a-z\d]+[\-_])*[a-z\d]+$/.test(accountId) &&
         accountId.length > 2 &&
```
Comment (ducnguyen-bitgo) [inline modules/account-lib/src/coin/near/utils.ts:51]: @facundoCarreSimtlix According to the near [doc](https://docs.near.org/docs/concepts/account), 2 is a valid account length so shouldn't it be `accountId.length >= 2`?

Comment (facundoCarreSimtlix) [reply modules/account-lib/src/coin/near/utils.ts:51]: yes, You are right, i fix that

**Review by ducnguyen-bitgo — APPROVED** [review]
Lgtm.

**Pull request merged**